README.md

```diff
@@ -243,6 +244,29 @@ Generatr container. So
 is needed to expose the container's port 8080 to the host (web browser). In the example above, the
 `-p 8080:8080` argument tells Docker to bind the local machine / host's port 8080 to the container's port 8080.
 
+#### Simplify using Docker-Compose
+Using Docker-Compose we can simplify the process.
+Add a docker-compose.yml next to your workspace.dsl.
+Add contents
+```
+version: '3'
+services:
+  structurizr-site-generatr:
+    image: ghcr.io/avisi-cloud/structurizr-site-generatr
+    volumes:
+      - .:/var/model
+    # command: generate-site -w workspace.dsl
```

Comment:
This line can be removed.

Reply:
I'd also maybe make it clear in the readme that you can edit the Docker Compose file to adjust the generate-site command with other switches (like adding the assets folder) and adjust the port where needed.